modules/fbx/SCsub

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+Import("env")
+Import("env_modules")
+
+env_fbx = env_modules.Clone()
+
+# Godot's own source files
+env_fbx.add_source_files(env.modules_sources, "tools/**.cpp")
+env_fbx.add_source_files(env.modules_sources, "data/**.cpp")
+env_fbx.add_source_files(env.modules_sources, "../../thirdparty/assimp_fbx/**.cpp")
+env_fbx.add_source_files(env.modules_sources, "**.cpp")
```

Comment:
I think a single wildcard works (`*.cpp`), no?

Reply:
Trying this out will let you know

Reply:
Works fine, wont work with removing the ../../ or by using # in the code, abs path wont work with * filename.